Framework/include/QualityControl/RepoPathUtils.h

```diff
@@ -128,6 +128,19 @@ class RepoPathUtils
 
   static constexpr auto allowedProvenancesMessage = R"(Allowed provenances are "qc" (real data processed synchronously), "qc_async" (real data processed asynchronously) and "qc_mc" (simulated data).)";
   static bool isProvenanceAllowed(const std::string& provenance);
+
+  static bool splitObjectPath(const std::string& fullPath, std::string& path, std::string& name)
```

Comment:
could you document that the 2nd and the 3rd arguments are outputs?

Otherwise, one could modify the function to return a tuple with `path` and `name`.

Reply:
This method must not be needed anymore if the change above is merged in